woocommerce/woocommerce-android
# [Scan to update inventory] Track additional analytics event

Closes: #10464 


### Description

This PR adds additional analytics event tracking when Quick inventory update bottomsheet is opened. 
More context – pdfdoF-4k6-p2#comment-5562

### Testing instructions

Go to the Products screen and click the "Scan" icon.
Verify the following events are tracked:
1. `🔵 Tracked: product_quick_inventory_update_bottom_sheet_shown, Properties: {"item_stock_managed":true}` - in case a stock-managed product was found
2. `🔵 Tracked: product_quick_inventory_update_bottom_sheet_shown, Properties: {"item_stock_managed":false}` - in case a non stock-managed product was found
